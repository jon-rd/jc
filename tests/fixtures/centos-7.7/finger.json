[{"login":"kbrazil","name":"kbrazil","tty":"tty1","idle":"1","login_time":"Apr  6 11:12","tty_writeable":true,"idle_minutes":1,"idle_hours":0,"idle_days":0,"total_idle_minutes":1},{"login":"kbrazil","name":"kbrazil","tty":"pts/0","idle":null,"login_time":"Apr  6 11:14","details":"(kbrazil-mac.attlocal.net)","tty_writeable":true,"idle_minutes":0,"idle_hours":0,"idle_days":0,"total_idle_minutes":0}]
