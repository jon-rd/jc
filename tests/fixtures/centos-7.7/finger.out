Login     Name       Tty      Idle  Login Time   Office     Office Phone   Host
kbrazil   kbrazil    tty1        1  Apr  6 11:12           
kbrazil   kbrazil    pts/0          Apr  6 11:14                           (kbrazil-mac.attlocal.net)
